.github/workflows/build-test.yml

```diff
@@ -239,8 +239,14 @@ jobs:
           rustup component add rust-src --toolchain nightly-2021-12-22
       - name: Install WASM tools
         run: |
-          sudo npm install -g wasm-opt --unsafe-perm
-          sudo npm install -g wabt
+          # This step requires wasm-opt and wabt. To install these two packages
+          # ready-to-use locally, you also can execute
+          # 'sudo npm install -g wasm-opt --unsafe-perm' and
+          # 'sudo npm install -g wabt'.
+          cd ffi/diplomat/wasm
+          npm ci
+          echo "$HOME/work/icu4x/icu4x/ffi/diplomat/wasm/node_modules/wasm-opt/bin" >> $GITHUB_PATH
```

Comment:
Issue: what is "$HOME/work/icu4x"?  Is that a GHA-specific thing?  Better would be to use "$PWD".

Reply:
Took this from the GHA docs, not sure it is GHA specific.
Addressed your concern in PR #1541.